src/lib.rs

```diff
@@ -486,4 +572,126 @@ mod test {
         assert_eq!(gas.clone().saturating_div(rhs), NearGas::from_gas(5));
         assert_eq!(gas.saturating_div(another_gas), NearGas::from_gas(0));
     }
+
+    #[test]
+    fn test_display() {
+        assert_eq!(NearGas::from_tgas(7).to_string(), "7 Tgas");
+        assert_eq!(NearGas::from_tgas(17).to_string(), "17 Tgas");
+        assert_eq!(NearGas::from_ggas(17).to_string(), "0.017 Tgas");
+        assert_eq!(NearGas::from_gas(17).to_string(), "<0.001 Tgas");
+        assert_eq!(NearGas::from_gas(0).to_string(), "0 Tgas");
```

Comment:
You may consider having an array of tuples like:

```suggestion
        for (gas, expected_display) in [
            (NearGas::from_tgas(7), "7 Tgas"),
            (NearGas::from_tgas(17), "17 Tgas"),
            ...
        ] {
            assert_eq!(gas.to_string(), expected_display);
        }
```

We may also extract it and then use to test parsing as well, and we should extensively test it for all the cases we handle, especially around ceiling logic ("1.999999 Tgas" should be displayed as "2 Tgas")